### ISSUE TITLE
placeholder.com discontinued

### ISSUE THREAD

**Issue by mbrodala** [IchHabRecht/filefill#74]
The known `placeholder.com` image service has been bought and replaced by pointless content:

![image](https://github.com/IchHabRecht/filefill/assets/5037116/2b27f590-429c-4c13-8719-eda308ef9ffe)

As per https://www.reddit.com/r/web_design/comments/10lqgja/placeholdercom_is_no_more/ there are a few alternatives, among them https://dummyimage.com and https://placehold.co where the latter seems to be a drop-in replacement.

**Comment by mbrodala** [comment]
Notice that the actual service at https://via.placeholder.com/ apparently still works, e.g. https://via.placeholder.com/100x100.png

Still it is questionable how long this is kept alive given that the main site has a completely different purpose now.

**Comment by mbrodala** [comment]
Sure, that's what the `imagebuilder` resource is for. This one uses the known `GifBuilder` internally:

https://github.com/IchHabRecht/filefill/blob/8f9c28688bc1d081fd2286b9c60c87173793cb85/Classes/Resource/Handler/ImageBuilderResource.php#L103

**Comment by kaystrobach** [comment]
Thank you - sry, missed it.

**Comment by andreaswolf** [comment]
via.placeholder.com seems to be gone by now, but it is still used as the default within PlaceholderResource.

**Comment by mbrodala** [comment]
Yup, https://github.com/IchHabRecht/filefill/pull/75 is still pending …

**Comment by mbrodala** [comment]
This is done by replacing `placeholder` with `placehold`, see https://github.com/IchHabRecht/filefill/commit/6502ae246278a5e8803cebcaeaa05276249cc308

**Comment by iamjaydatt** [comment]
> Please find your information in the Readme https://github.com/IchHabRecht/filefill#image-builder

img.itisuniqueofficial.com – The Ultimate Image Placeholder Generator

Looking for a faster, cleaner, and more powerful placeholder generator than the rest? Try img.itisuniqueofficial.com – lightweight, super fast, and easy to use for developers, bloggers, and designers.

* Generate placeholders in any size and format
* Customize background, text, and colors
* Works instantly with simple URLs
* Faster than other placeholder tools

Learn how to use it here: [https://img.itisuniqueofficial.com/documentation.html](https://img.itisuniqueofficial.com/documentation.html)
